src/platform/3ds/3ds_ui.cpp

```diff
@@ -120,6 +116,13 @@ CtrUi::CtrUi(int width, int height, const Game_ConfigVideo& cfg) : BaseUi(cfg)
 	top_image.tex = tex;
 	top_image.subtex = &subt3x;
 
+	// FIXME: Add option to settings Ui
+	if (cfg.stretch_width.Get()) {
```

Comment:
This has nearly no effect, since you disable stretching in `UpdateDisplay()`.
The idea here was to handle this like fullscreen for other platforms. So, no real need for config option here. Same is true for switch and vita, where you only made this a FIXME without option.

Reply:
Yeah this change makes no sense currently because there must be a function for toggling this. This is similiar to a "keep aspect ratio" setting. Could be even provided for SDL2 in case somebody likes it to have everything stretched... (that's why it is not the same as fullscreen)

Wanted to finish this as part of the Settings PR as I cannot do it in this PR properly (And breaking that feature for a while isn't toooo bad)